### DISCUSSION TITLE
Adding issues to your forked Repository

### DISCUSSION THREAD

**Discussion by nahume24** [community/community#33911] [Projects and Issues] [answered]
I want to emerge two repos one I created and one that I have forked. I have clone my original and tried to merge with the repo but I cant seem to get it right.  I have forked a repository and rename it. Now I want to add an issue but there is no tab for issues.

**Comment by airtower-luna** [comment, marked as answer]
Check the repository settings. You can enable/disable issues there, and for forks they're disabled by default:

![image](https://user-images.githubusercontent.com/5814919/191995567-29297e27-29d3-4a16-983b-1c3e35b7682f.png)
